roles/monitoring/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
+# tasks file for monitoring
+
+  - name: Install Logwatch
+    apt: 
+      pkg: logwatch 
+      state: installed
+
+  - name: Configure Logwatch
+    command: logwatch --detail Medium --mailto {{ Email }} --output mail --service All --range 'between -7 days and today'  
+     
+# Installation of chkrootkit
+
+  - name: Install chkrootkit
+    apt:
+      pkg: chkrootkit
```

Comment:
ola afta ta ksexwrista installation, mporeis na ta kaneis ena task me to with_items